<ui version="4.0" >
 <class>LookupCertificatesDialog</class>
 <widget class="QDialog" name="LookupCertificatesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Keyserver Certificate Lookup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="1" >
      <widget class="KLineEdit" name="findED" />
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="findPB" >
       <property name="text" >
        <string>Search</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3" >
      <widget class="Line" name="hr" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="2" >
      <widget class="QPushButton" name="detailsPB" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Details...</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="6" column="2" >
      <widget class="QPushButton" name="saveAsPB" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Save As...</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QPushButton" name="selectAllPB" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Select All</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2" >
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2" >
      <widget class="QPushButton" name="deselectAllPB" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Deselect All</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="findLB" >
       <property name="text" >
        <string>Find:</string>
       </property>
       <property name="buddy" >
        <cstring>findED</cstring>
       </property>
      </widget>
     </item>
     <item rowspan="6" row="2" column="0" colspan="2" >
      <widget class="QTreeView" name="resultTV" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3" >
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global" >KLineEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>findED</sender>
   <signal>returnPressed()</signal>
   <receiver>findPB</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>409</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel" >
     <x>430</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LookupCertificatesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>28</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel" >
     <x>46</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LookupCertificatesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>188</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel" >
     <x>218</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectAllPB</sender>
   <signal>clicked()</signal>
   <receiver>resultTV</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>437</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel" >
     <x>409</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deselectAllPB</sender>
   <signal>clicked()</signal>
   <receiver>resultTV</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>465</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel" >
     <x>409</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findPB</sender>
   <signal>clicked()</signal>
   <receiver>LookupCertificatesDialog</receiver>
   <slot>slotSearchClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>437</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel" >
     <x>443</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>detailsPB</sender>
   <signal>clicked()</signal>
   <receiver>LookupCertificatesDialog</receiver>
   <slot>slotDetailsClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>523</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel" >
     <x>541</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAsPB</sender>
   <signal>clicked()</signal>
   <receiver>LookupCertificatesDialog</receiver>
   <slot>slotSaveAsClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>517</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel" >
     <x>539</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findED</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LookupCertificatesDialog</receiver>
   <slot>slotSearchTextChanged()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>93</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel" >
     <x>93</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSearchClicked()</slot>
  <slot>slotDetailsClicked()</slot>
  <slot>slotSaveAsClicked()</slot>
  <slot>slotSearchTextChanged()</slot>
 </slots>
</ui>
